eultheme/middleware.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.shortcuts import render
+from downtime.middleware import DowntimeMiddleware
+
+class DownpageMiddleware(DowntimeMiddleware):
+
+    def process_request(self, request):
+        def get_client_ip(request):
+            x_forwarded_for = request.META.get('HTTP_X_FORWARDED_FOR')
```

Comment:
My understanding is that HTTP_X_FORWARDED_FOR is easier to fake, so I wonder about not using this one if we don't need it.  If we do decide to keep it (probably not such a huge risk in this case), please document the splitting you're doing (I'd forgotten - or maybe never knew - that it could have multiple hosts).